### DISCUSSION TITLE
"show sort headers in all views" - now to enable WITHOUT running installer?

### DISCUSSION THREAD

**Discussion by FredThompsonII** [Open-Shell/Open-Shell-Menu#700] [Q&A]
I have almost all of Open-Shell-Menu working properly by copying the appropriate files to a folder, ProgramData, AppData, and Windows directories. There is also a large reg file that is merged.

This is because I don't want Open-Shell-Menu's entry in the install/remove programs list and I've converted almost everything I use into portable or scripted "hooking" into Windows. (Long story...this is easier than manually reinstalling every time there's a big problem...family members...most effective to do this and rerun the "hooking" routines on every boot...)

Everything works EXCEPT the sort headers will not display.

Attached are the batch and reg files I am using.
[Open-Shell - Configure.zip](https://github.com/Open-Shell/Open-Shell-Menu/files/6393065/Open-Shell.-.Configure.zip)

The "meat" of the installation, after files have been placed and registry settings...set is:

`gpupdate /force - this fails, don't know why...

taskkill /im explorer.exe /f
start explorer.exe
start "" /d "%systemdrive%\Portable\- Linked\System\Open-Shell\" /i /normal StartMenu.exe
`

What should I add to my "hooking" method so this setting will work?

**Comment by Yaron10** [comment]
@FredThompsonII,

This is NOT a reply to your question.
Could you please have a look at https://github.com/Open-Shell/Open-Shell-Menu/discussions/753?

I'd appreciate your help.
